build/pmm-server-docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM oraclelinux:7
```

Comment:
I suggest using slim image. It can help minimize image size

Reply:
Yes, we'll use `slim`. I just want to start with regular and then switch to slim to solve fewer problems in one step

Reply:
heh, here difference between slip and regular:
```
perconalab/pmm-server-fb                                PR-2519-c2d9644   5d08e5a0a214   18 minutes ago      1.76GB
perconalab/pmm-server-fb                                PR-2519-f0394b3   8e1be47a9e6f   About an hour ago   1.78GB
```